frontend/components/Modal/_styles.scss

```diff
@@ -26,7 +26,7 @@
     margin-top: $pad-large;
     font-size: $x-small;
     max-height: 800px;
-    overflow-y: auto;
+    overflow: visible;
```

Comment:
Allows dropdown select menus to display outside of short modals (like transfer hosts).